web/portal/platform/src/translations/es.json

```diff
@@ -114,6 +118,8 @@
   "Global Special Number_one": "Número especial Global",
   "Global Special Number_other": "Números especiales Globales",
   "Go to brands": "Ir a marcas",
+  "Grant Read Only Access": "Acceso de solo lectura",
+  "Grant Write Access": "Acceso de escriptura",
```

Comment:
escritura

Reply:
You have included this platform portal change in a commit tagged as portal/brand